tk/internal/reducer/reducer.go

```diff
@@ -590,7 +628,9 @@ func (r *Reducer) applyTaskAttachmentRemove(e types.Event) error {
 
 	task, ok := r.tasks[payload.TaskUUID]
 	if !ok {
-		return fmt.Errorf("task UUID not found: %s", payload.TaskUUID)
+		// Task was deleted (likely via project.delete cascade)
+		// Skip orphaned event silently - this is expected for old data
```

Comment:
No, no, no, no, let's not do that. Either we don't have any such events, and then everything is okay. Or we do have such events, then I would like to know about them.